kymatio/scattering3d/scattering3d.py

```diff
@@ -273,9 +273,9 @@ def forward(self, input_array, order_2=True, rotation_covariant=True,
         ----------
         input_array: torch tensor 
             input of size (batchsize, M, N, O)
-        order_2: bool, optional
-            if set to False|True it also excludes|includes second order
-            scattering coefficients (default: True).
+        max_order: int, optional
+            The maximum order of scattering coefficients to compute. Must be
+            either `1` or `2`. Defaults to `2`.
```

Comment:
These numbers have backticks, below they don't.
I guess it renders nice in the docs but for those looking at docstrings it adds visual noise.

In principle I'm fine either way, but I think that it should be at least roughly consistent with below. And I think making it consistent might be an annoying task, so removing the backticks might be easier :)

Reply:
Again, see above. Will remove the backticks here and make a separate PR to clean up the docstrings.